e2e/various-suite/exemplars.spec.ts

```diff
@@ -10,7 +10,7 @@ const addDataSource = () => {
       e2e.components.DataSource.Prometheus.configPage.exemplarsAddButton().click();
       e2e.components.DataSource.Prometheus.configPage.internalLinkSwitch().check({ force: true });
       e2e.components.DataSource.DataSourceHttpSettings.urlInput().type('http://prom-url:9090');
-      e2e.components.DataSourcePicker.inputV2().should('be.visible').click({ force: true });
+      e2e.components.DataSourcePicker.inputV2().click({ force: true }).should('have.focus');
```

Comment:
Is there a reason why "should be visible" part removed?

Reply:
This unit test used to be breaking on this branch without this fix, however I was just able to revert it back and now it's fine, so I'll revert this and hope it drone passes 🤞

Reply:
🤞

Reply:
While this test passes for me locally, it's still failing drone. I'm putting this change back in

Reply:
so weird, it's passing again after reintroducing